pages/create/index.page.js

```diff
@@ -1,72 +1,113 @@
+import { number, object, string, date, ref } from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+
 import Button from "../../components/button/button";
 import Input from "../../components/input/input";
 import Select from "../../components/select/select";
-import { Container, Form, Label, Title } from "./create.styles";
-import { useForm } from "react-hook-form";
-import { number, object, string, date, ref } from "yup";
-import { yupResolver } from "@hookform/resolvers/yup";
+
+import { Container, Form, Title } from "./create.styles";
 
 const Create = () => {
   const validationSchema = object().shape({
-    projectName: string().required(),
+    title: string().required(),
     subtitle: string().required(),
-    category: object().required(),
-    description: string().required(),
-    currency: object().required(),
+    story: string().required(),
+    token: string().required(),
     fundGoal: number().required(),
     startDate: date()
       .required()
-      .min(new Date(), "Created date can not be future"),
+      .max(new Date(), "Created date can not be future"),
     endDate: date()
       .required()
       .min(ref("startDate"), "End date should be after start date"),
   });
 
-  const {
-    handleSubmit,
-    reset,
-    formState: { errors },
-  } = useForm({
+  const formOptions = {
     resolver: yupResolver(validationSchema),
     defaultValues: {},
     mode: "onChange",
-  });
+  };
+  const { handleSubmit, reset, register, formState } = useForm(formOptions);
+  const { errors } = formState;
 
   const onSubmit = (data) => {
-    console.log("POST", data);
+    console.log("POST:", data);
```

Comment:
Suggestion: add a logger library instead of using console.log 
That way we can check the logs on the server and not into the console of the browser